### ISSUE TITLE
Unable to recover password

### ISSUE THREAD

**Issue by munoz-romero** [netlify/netlify-identity-widget#123]
We have been using this service and widget since November, but in the last few weeks we have noticed three different cases when users try to recover password:

1.- Sometimes recover email doesn't get to email inbox.
2.- Some users try to login with Google credentials, but it keeps failing. Then, they tried to sign up (with the same email id), and they got the message that account already exists. So they tried to reset the password through ‘Forgot password’ link, and that too doesn’t work.
3.- When recover password email is received, the link to recover it opens web app correctly, but a message "The user doesn't exist" appears in the identity widget.

Our web app is https://app.naiz.chat with netlify-identity-widget 1.4.11

**Comment by jlengstorf** [comment]
we've shipped multiple updates that should improve this process. I haven't heard recent reports of issues like this, so I'm going to close. however, if you hit this issue, please open a new issue! if possible, include a reproduction or screen recording so we can see what's going on and dig into a fix

thanks!

**Comment by skllpr** [comment]
I still have this issue. After clicking on the email, the redirect back to the identity widget just logs in the user.

**Comment by skllpr** [comment]
So there are several cases of this issue:

1.) Attempted password recovery to change password after user is logged in, user logs out and then clicks link through email
2.) Repeated attempts to recover password (only 1 email is sent, reuse of that email causes user not found issue)

There seem to be several more use-cases but I am having difficulty identifying them.

**Comment by skllpr** [comment]
Basically it works only on the first try of the password recovery. A second attempt to change password fails. This seems to be a pretty high priority issue (if users are being effected).

To recreate issue, try forgot password, change password, try forgot password again, it will fail.

**Comment by skllpr** [comment]
Yeah this issue is still present, I believe this should be reopened.

**Comment by nunocompleto** [comment]
Please I'm having this issue(not getting the reset password email) and thus unable to log in into my account. (been trying for 2 days and yet not in my inbox, spam or anywhere at all)

**Comment by aentwist** [comment]
@jlengstorf please reopen this, there has been enough information added. Additionally, I will build on this with further clarification.

- If the user is logged in then the recovery state is not reached
- If the user logs out and clicks the recovery link again it automatically goes to state "Updating password" followed by "User not found"
- When the user tries the "Forgot password" process on their end no email is sent (potentially due to "A Netlify Identity cannot [...]" (see below)

- A Netlify Identity cannot have recovery triggered from the Netlify UI more than once (e.g. to generate a new recovery token / link)
- The Netlify UI does not allow admins to set the passwords of the users manually

=> the whole account is royally screwed and a new account must be created

**Comment by aentwist** [comment]
For the two Netlify Identity service issues I created a [Support Forum thread](https://answers.netlify.com/t/bug-report-netlify-identity-password-is-unrecoverable-if-it-fails-the-first-time/102646). The other issues need addressed here.

**Comment by jlengstorf** [comment]
hey, y'all — I'm no longer at Netlify, so I can't be much help here. I'll reopen this issue so the Netlify team can track it, but after that I'm going to unsub from this issue. thanks to everyone who has provided extra detail!
